build-system/tasks/check-links.js

```diff
@@ -166,13 +166,18 @@ function filterWhitelistedLinks(markdown) {
  * @param {string} markdownFile Path of markdown file, relative to src root.
  * @return {Promise} Used to wait until the async link checker is done.
  */
-function runLinkChecker(markdownFile) {
+async function runLinkChecker(markdownFile) {
+  // The extension generator markdown template has placeholder links that can't
+  // be checked.
+  if (/doc-template\.md$/.test(markdownFile)) {
+    return;
+  }
```

Comment:
If the markdown template file isn't ever supposed to be rendered as HTML, maybe it should be a `.txt` file and not an`.md` file? (We're likely to introduce other checks for `.md` files, at which point this file will needlessly be flagged.)

Reply:
Hmm, I think keeping `md` is more ergonomic for the outreach wg to edit this since they'd get proper tooling (e.g. syntax highlighting, snippets) with the correct extension by default.

We can also make this rule more generic by using a standard suffix like `-TEMPLATE`.

Reply:
I support keeping it as an `md` file, maybe using the convertion `foo.template.md` which iirc is a convention used in Angular

Reply:
Keeping `.md` SGTM. (I'm in the process of standardizing / auto-fixing all markdown files in the repo, and this file can benefit from that.)

Reply:
`.template.md` sgtm.

Reply:
**Update:** Glad you went with `.md`. I've implemented markdown file auto-formatting in #25182, and the template file was cleaned up. I've tested `gulp make-extension` to ensure that all interpolation is still being done correctly.